src/js/utils/use-isomorphic-layout-effect.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-restricted-imports */
+import React, { useEffect } from 'react';
```

Comment:
Is it possible not to import the whole `React`? Like `import { useEffect, useLayoutEffect } from 'react';`

Reply:
Will change this.